transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -50,26 +50,104 @@ public void addNewEvent(Event event) {
 
     @Override
     protected void run() {
-        //Todo
         for (;;) {
-            // wait until an event has finished
-            //final long cqe = Native.ioUringWaitCqe(io_uring);
-            //final Event event = events.get(Native.ioUringGetEventId(cqe));
-            //final int ret = Native.ioUringGetRes(cqe);
-            // switch (event.getOp()) {
-            //     case ACCEPT:
-            //         // serverChannel is necessary to call newChildchannel
-            //         // create a new accept event
-            //         break;
-            //     case READ:
-            //         // need to save the Bytebuf before I execute the read operation
-            //         // fireChannelRead(byteBuf)
-            //         break;
-            //     case WRITE:
-            //         // you have to store Bytebuf to continue writing
-            //         break;
-            // }
-            // processing Tasks
+            final IOUringCompletionQueue ioUringCompletionQueue = ringBuffer.getIoUringCompletionQueue();
+            final IOUringCqe ioUringCqe = ioUringCompletionQueue.peek(); // or waiting
+
+            if (ioUringCqe != null) {
+                final Event event = events.get(ioUringCqe.getEventId());
+                System.out.println("Completion EventId: " + ioUringCqe.getEventId());
```

Comment:
All `System.out.println(Object)` should be removed.

Reply:
it's only temporary, it is easier to debug

Reply:
Oh okay, makes sense.